CMakeLists.txt

```diff
@@ -57,6 +58,13 @@ if("${CMAKE_BUILD_TYPE}" STREQUAL "")
 	set(CMAKE_BUILD_TYPE Debug)
 endif()
 
+# Hot reload is enabled by default in Debug-builds
+if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
```

Comment:
This is fine, given that it looks like "Debug" and "Release" are the only options here. With scons, there's also the possibility of making editor builds, which is why the logic there is "not release" as opposed to checking directly for debug. I guess if we ever have editor builds via cmake, though, we'll need to change this.